client/src/components/EditDetails.jsx

```diff
@@ -1,60 +1,68 @@
-import React from 'react';
-import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
-import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css';
-import axios from 'axios';
+import React from 'react'
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap'
+import '!style-loader!css-loader!bootstrap/dist/css/bootstrap.css'
+import axios from 'axios'
+
+const Months = Array.from(Array(12)).map(i => i + 1)
+const Days = Array.from(Array(31)).map(i => i + 1)
 
 class EditDetails extends React.Component {
   constructor(props) {
-    super(props);
-    let { height, weight, age, gender, dob, zip } = props.profile;
+    super(props)
+    let { height, weight, age, gender, dob, zip } = props.profile
+
+    const [heightFt, heightIn] = height.split('/')
+
     this.state = {
-      heightFt: height.split('/')[0],
-      heightIn: height.split('/')[1],
-      weight: weight.split(' ')[0],
       age,
-      gender,
       dob,
```

Comment:
generally try to avoid abbreviations as much as possible, since it reduces readability for all engineers (including your future self), and takes more mental power than necessary to understand what this variable stands for. Prefer `dateOfBirth` over `dob`